pallets/loans/src/migrations.rs

```diff
@@ -59,7 +67,32 @@ pub mod v3 {
         Markets::<T>::iter().for_each(|(asset_id, _)| {
             log::info!("market {:#?} need to migrate", asset_id,);
         });
-        log::info!("👜 loans borrow-limit migration passes PRE migrate checks ✅",);
+        let reward_speed_count = MarketRewardSpeed::<T>::iter().count();
+        log::info!(
+            "total {:#?} reward speed items need to migrate",
+            reward_speed_count
+        );
+
+        let reward_accrued_count = RewardAccured::<T>::iter().count();
+        log::info!(
+            "total {:#?} reward accrued items need to migrate",
+            reward_accrued_count
+        );
+        let mut reward_accrued_count = 0;
+        RewardAccured::<T>::iter().for_each(|(account_id, reward_accrued)| {
```

Comment:
how many accounts do we currently have on live, if it's too big maybe will break the chain I guess

Reply:
@MrPai we have 174 items, do you think it's fine?

Reply:
174 items will be fine.